projects/snappy/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://github.com/google/snappy"
+primary_contact: "victor@costan.us"
```

Comment:
@pwnall please check the contact here

Reply:
`storage-dev@chromium.org` would be better